<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerManagmentWidget</class>
 <widget class="QWidget" name="ControllerManagmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Controller</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="7" column="8">
       <widget class="QPushButton" name="showGraph">
        <property name="text">
         <string>Graph</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="convertToCuboid">
        <property name="text">
         <string>Cuboid</string>
        </property>
        <property name="icon">
         <iconset resource="../Workspace.qrc">
          <normaloff>:/Workspace/Resources/icons/cuboid.png</normaloff>:/Workspace/Resources/icons/cuboid.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="7" colspan="2">
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="convertGcAxis">
        <property name="maximum">
         <number>2</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="cuboidMethod">
        <item>
         <property name="text">
          <string>MinOBB3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OBB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OBB2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="7" colspan="2">
       <widget class="QPushButton" name="resetButton">
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 85, 85);
font: italic;
</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="8">
       <widget class="QCheckBox" name="showPrimitives">
        <property name="text">
         <string>Show primitives</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="convertToGC">
        <property name="text">
         <string>GC</string>
        </property>
        <property name="icon">
         <iconset resource="../Workspace.qrc">
          <normaloff>:/Workspace/Resources/icons/gc.png</normaloff>:/Workspace/Resources/icons/gc.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="6">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Gaussian sigma</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="skeletonJoints">
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QCheckBox" name="basicFitGC">
        <property name="text">
         <string>Basic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Delete prim</string>
        </property>
        <property name="icon">
         <iconset resource="../Workspace.qrc">
          <normaloff>:/Workspace/Resources/icons/remove.png</normaloff>:/Workspace/Resources/icons/remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="7" colspan="2">
       <widget class="QSlider" name="gaussianSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QCheckBox" name="useAABB">
        <property name="text">
         <string>useAABB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Workspace.qrc"/>
 </resources>
 <connections/>
</ui>
